src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -8,60 +8,52 @@
 
 #include <ShellScalingApi.h>
 #include <mutex>
+#include <fileapi.h>
 
 #include <gdiplus.h>
 
 namespace ZoneWindowUtils
 {
-    const std::wstring& GetActiveZoneSetTmpPath()
+    const wchar_t ActiveZoneSetsTmpFileName[] = L"FancyZonesActiveZoneSets.json";
+    const wchar_t AppliedZoneSetsTmpFileName[] = L"FancyZonesAppliedZoneSets.json";
+    const wchar_t DeletedCustomZoneSetsTmpFileName[] = L"FancyZonesDeletedCustomZoneSets.json";

```

Comment:
Add a comment to clarify these are not localizable strings.

Reply:
Done